content/docs/fundamentals/converting/resolving-errors@ja.md

```diff
@@ -0,0 +1,348 @@
+
+AMP のレイアウト システムでは、ページ上の要素をさまざまな方法（サイズの固定、レスポンシブ デザイン、高さの固定など）で配置し、拡大縮小できます。
+
+この記事のケースでは、レイアウト システムは `amp-img` のレイアウト タイプを `container` タイプと推測しました。しかし、`container` タイプは、子要素を含む要素にのみ適用可能です。`container` タイプは `amp-img` タグに対応していないため、このエラーの原因となりました。
+
+なぜ `container` タイプが推測されたのでしょうか。その理由は、`amp-img` タグに `height` 属性を指定していなかったためです。HTML では、ページ上の要素に固定の幅と高さを常に指定することで、リフローを削減できます。AMP では、AMP が要素のアスペクト比を事前に判断できるように、amp-img 要素の幅と高さを定義する必要があります。
+
+次のように、`<amg-img>` タグに `width` と `height` を**追加**してください。
+
+```html
+<amp-img src="mountains.jpg" width="266" height="150"></amp-img>
+```
+
+ページを更新して、検証ツールで確認してみましょう。エラーが表示されなくなるはずです。
+
+これで有効な AMP ドキュメントになりましたが、ページ上の配置が不自然なため、画像の見栄えがよくありません。デフォルトでは、`amp-img` の高さと幅を指定した場合、AMP は指定に合わせてサイズを修正します。しかし、どのような画面サイズでも、AMP が画像を*レスポンシブ*に拡大縮小してページに合わせられたら、最適でしょう。
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-not-responsive.png', 412, 660, align='center third', caption="この画像はレスポンシブではない")}}
+
+AMP では、指定された幅と高さから要素のアスペクト比を算出できます。それにより、AMP レイアウト システムはさまざまな方法で要素を配置し、拡大縮小できます。`layout` 属性は、要素をどのように配置し、拡大縮小するかを AMP に伝えます。
+
+画像の拡大縮小とサイズ変更が行われるように、layout 属性を `responsive` に**設定**しましょう。
+
+```html
+<amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+```
+
+これで、画像が正しいアスペクト比になり、画面の幅にレスポンシブに収まりました。
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-responsive.png', 412, 660, align='center third', caption="画像がレスポンシブになった") }}
+
+{% call callout('詳細情報', type='read') %}
+AMP レイアウト システムについて詳しくは、[AMP レイアウト仕様](/ja/docs/design/amp-html-layout.html)をご覧ください。
+{% endcall %}
+
+## 完了
+
+AMP ドキュメントは次のようになります。
+
+```html
+<!doctype html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+    <link rel="canonical" href="/article.html">
+    <link rel="shortcut icon" href="amp_favicon.png">
+
+    <title>News Article</title>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        width: auto;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        background: Tomato;
+        color: white;
+        font-size: 2em;
+        text-align: center;
+      }
+
+#h01 
```

Comment:
Correct line 303 ... it should just be ` h1 {`

Note: Code should never be altered.